redhat-developer/developers.redhat.com
# [RHDX-101] Fix code snippet assembly

### JIRA Issue Link
[RHDX-101](https://projects.engineering.redhat.com/browse/RHDX-101)

Affects 4 instances of content using the Code snippet assembly

1. https://developer-preview-3449.ext.us-west.dc.preprod.paas.redhat.com/che-custom-stacks/ - View steps 3, 4 and 5
2. https://developer-preview-3449.ext.us-west.dc.preprod.paas.redhat.com/coderland/serverless/building-a-serverless-service/ - Find mvn clean package
3. https://developer-preview-3449.ext.us-west.dc.preprod.paas.redhat.com/coderland/serverless/deploying-serverless-knative/ - Find minishift config set openshift-version v3.11.0
4. https://developer-preview-3449.ext.us-west.dc.preprod.paas.redhat.com/products/fuse/hello-world-fuse-tooling/ - The first content on the page is a snippet, see oc new-project knativetutorial


### Verification Process
1. View the current content that has code snippet assemblies and verify they appear in the PR environment, compared to production where the snippet does not render on the page.
2. Create test content and create various code snippets, assigning language values or leaving the values empty.  Verify the Code* field of the assembly is required, and that you cannot save it with no content.
3. Verify (if possible) that the highlight.js js and styles are inherited from the rhd-frontend site. 
The rhd-frontend theme will show the language syntax highlighting with the github style.
4. Example #2 above has a code snippet using visual style no-padding-top and no-padding-bottom. 

Notes:
- I did not test it with **audience selection** and I assume these fields still function as expected. 
- The name and author of the snippet do not appear on the site.
